src/main/java/org/spongepowered/api/attributes/Attribute.java

```diff
@@ -0,0 +1,36 @@
+package org.spongepowered.api.attributes;
+
+public interface Attribute {
+	
+	/**
+	 * Gets the ID of this attribute as a string.
+	 * @return The ID of this attribute as a string.
```

Comment:
Also, it is common in the API that a public interface does not denote the provided methods as `public`. So, for conformity, please remove the `public` modifier to the methods.